packages/eslint-plugin-pf-codemods/src/rules/v6/formFiledGroupHeaderTitleTextObjectRenamed/formFiledGroupHeaderTitleTextObject-renamed.test.ts

```diff
@@ -39,5 +39,19 @@ ruleTester.run("formFiledGroupHeaderTitleTextObject-renamed", rule, {
         },
       ],
     },
+    {
+      code: `import { FormFiledGroupHeaderTitleTextObject } from '@patternfly/react-core'; export { FormFiledGroupHeaderTitleTextObject as TitleTextObject }`,
+      output: `import { FormFieldGroupHeaderTitleTextObject } from '@patternfly/react-core'; export { FormFieldGroupHeaderTitleTextObject as TitleTextObject }`,
```

Comment:
Related to above, for the tests we want the invalid case to be an export `from @patternfly....`, while a valid case would actually be an export like you have here (not from patternfly)

Reply:
So if I understand, just this:
```
export { FormFiledGroupHeaderTitleTextObject as TitleTextObject }
```
should be valid.

But when the import is included:
```
import { FormFiledGroupHeaderTitleTextObject } from "@patternfly/react-core";
export { FormFiledGroupHeaderTitleTextObject as TitleTextObject };
```
shouldn't that be invalid? Because if the unaliased import is present, then we know the exported name is 100% the same name we import from Patternfly.